src/components/browse/components/SearchResultTable.js

```diff
@@ -898,26 +903,35 @@ class DimensioningContainer extends React.PureComponent {
     }
 
     render(){
-        const { columnDefinitions, windowWidth, isOwnPage, maxHeight = 500 } = this.props;
-        const { results, tableContainerWidth, tableContainerScrollLeft, mounted, widths, openDetailPanes, tableLeftOffset } = this.state;
+        const {
+            columnDefinitions,
+            windowWidth,
+            context, // May be page context or virtualContext
+            isOwnPage = true,
+            navigate,
+            rowHeight = 47, // Must be aligned in CSS stylesheets
```

Comment:
Eventually we could just set height: 47 (unless open detail pane) on ResultRow somewhere and have rowHeight be fully controllable from JS... idk for future consideration.